### PR TITLE
Feature - licensing ppl updates

### PR DESCRIPTION
**Pull request by joefitter**
* add changedBy meta to project fork tasks so we can record whether the version was forked by ASRU
* Added check to determine if the current version can be updated by the logged in user

### CODE REVIEW

**Comment by lennym** [comment]
Does this still allow ASRU edited versions, which have been granted, to then be further amended by  establishment users?

File: lib/routers/establishment/project-versions.js [hunk 1]
```diff
@@ -112,16 +119,19 @@ router.put('/:id/:action',
   perms('project.update'),
   validateAction,
   canUpdate,
+  userCanUpdateVersion,
```
Comment (lennym) [inline lib/routers/establishment/project-versions.js]: Specifically here, which I _think_ is the endpoint to perform a `fork` operation and start an amendment.

Comment (lennym) [reply lib/routers/establishment/project-versions.js]: Ahhh no. Not here.

Nothing to see here...

Comment (joefitter) [reply lib/routers/establishment/project-versions.js]: it _should_ be an endpoint on the project

**Pull request merged**